Client/public/locales/en/student.json

```
@@ -0,0 +1,10 @@
+{
+    "student": {
+        "1": "Filter to Find Supporter",
```

I think we should use more descriptive names here than numbers
```suggestion
        "findSupporter": "Filter to Find Supporter",
```

---

Fixed